string[] symbols
---
kineverse/Constraint[] constraints